docs/_articles/en/v3.x/salesforce-graph-engine/rules.md

```diff
@@ -152,63 +186,23 @@ To run the path-based analysis rules run `scanner:run:dfa`. Alternatively, run `
 | **Sanitizer**	 |				                                  														      		|
 |                |Non-null initialization of variables and null checks before accessing. Examples: `String s = 'abcde';` , `if (s != null) {` |
 
-### Interpreting ApexNullPointerExceptionRule Results
+#### Interpreting ApexNullPointerExceptionRule Results
 
 Match any violation message that you receive with this case to understand more about the violation.
 
 *Common Case*
 
 > ApexNullPointerExceptionRule identifies Apex operations with a high likelihood of throwing a NullPointerException
 
-Parameter explanation:
+Explanation:
 
 The operation dereferences a null object and throws a NullPointerException. Review your code and add a null check.
 
-## UnimplementedTypeRule
+### AvoidMultipleMassSchemaLookups Rule
```

Comment:
This shouldn't end in "Rule", also it needs an anchor:
`### AvoidMultipleMassSchemaLookups <a name='AvoidMultipleMassSchemaLookups'>#</a>`

Reply:
@jfeingold35 Why shouldn't it end in Rule? I added rule for the two rules whose names don't include "rule".